examples/tensorflow/image_recognition/keras_models/mnist/quantization/qat/main.py

```diff
@@ -0,0 +1,179 @@
+    if FLAGS.benchmark and FLAGS.mode == 'performance':
+        iteration = 100
+    postprocess = LabelShift(label_shift=1)
+    metric = TensorflowTopK(k=1)
+
+    def eval_func(dataloader):
+        latency_list = []
+        for idx, (inputs, labels) in enumerate(dataloader):
+            # dataloader should keep the order and len of inputs same with input_tensor
+            assert len(input_tensor) == len(inputs), \
+                'inputs len must equal with input_tensor'
+            feed_dict = dict(zip(input_tensor, inputs))
+
+            start = time.time()
+            predictions = model.sess.run(output_tensor, feed_dict)
+            end = time.time()
+
+            predictions, labels = postprocess((predictions, labels))
+            metric.update(predictions, labels)
+            latency_list.append(end-start)
+            if idx + 1 == iteration:
+                break
+        latency = np.array(latency_list).mean() / FLAGS.batch_size
+        return latency
+
+    dataloader = dataloader(batch_size=FLAGS.batch_size)
+    latency = eval_func(dataloader)
+    if FLAGS.benchmark and FLAGS.mode == 'performance':
+        print("Batch size = {}".format(FLAGS.batch_size))
+        print("Latency: {:.3f} ms".format(latency * 1000))
+        print("Throughput: {:.3f} images/sec".format(1. / latency))
+    acc = metric.result()
+    return acc 
+
+
+def main():
+    if FLAGS.tune:
+        logger.info('start quantizating the model...')
+        from neural_compressor import training, QuantizationAwareTrainingConfig
+        config = QuantizationAwareTrainingConfig()
+        compression_manager = training.prepare_compression(FLAGS.input_model, config)
+        compression_manager.callbacks.on_train_begin()
+
+        q_aware_model = compression_manager.model.model
+
+        q_aware_model.compile(optimizer='adam',
+                    loss=tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True),
+                    metrics=['accuracy'])
+
+        q_aware_model.summary()
+        train_images_subset = train_images[0:1000]
+        train_labels_subset = train_labels[0:1000]
+        q_aware_model.fit(train_images_subset, train_labels_subset,
+                        batch_size=500, epochs=1, validation_split=0.1)
+        _, q_aware_model_accuracy = q_aware_model.evaluate(
+                                        test_images, test_labels, verbose=0)
+        print('Quant test accuracy:', q_aware_model_accuracy)
+        
+        compression_manager.callbacks.on_train_end()
+        compression_manager.save(FLAGS.output_model)
+
+    if FLAGS.benchmark:
+        from neural_compressor.benchmark import fit
+        from neural_compressor.experimental import common
+        from neural_compressor.config import BenchmarkConfig
+        assert FLAGS.mode == 'performance' or FLAGS.mode == 'accuracy', \
+        "Benchmark only supports performance or accuracy mode."
+
+        model = common.Model(FLAGS.input_model).graph_def
```

Comment:
common.Model needs to be updated after https://github.com/intel/neural-compressor/pull/244 merged.